### ISSUE TITLE
[Project] Our Radiant Future

### ISSUE THREAD

**Issue by xeophin** [jsoma/data-studio-projects#222]
Please complete all of the following sections, or the ghost of Joseph Pulitzer will spookily dance around your issue! A completed version of this template can be found at https://github.com/jsoma/data-studio-projects/issues/1

# Pitch

## Summary

With more and more devices coming online and communicating with the wider world via radio waves, I would like to have a look at the cell phone towers in Switzerland, their distribution and broadcasting power and see whether I can draw a map of where you're getting cooked – or where you will find the last dead zone in Switzerland.

## Details

**Possible headline(s):** This is the last dead zone in Switzerland

**Data set(s):** https://map.geo.admin.ch/?topic=funksender&lang=de&bgLayer=ch.swisstopo.swissimage&layers=ch.bakom.radio-fernsehsender,ch.bakom.mobil-antennenstandorte-gsm,ch.bakom.mobil-antennenstandorte-umts,ch.bakom.mobil-antennenstandorte-lte&catalogNodes=403,408&layers_visibility=false,true,true,true&zoom=2&E=2670802.96&N=1236935.32

**Code repository:** https://github.com/xeophin/lede-data-studio/tree/master/code/02-antennas
	
**Possible problems/fears/questions:** 
- Broadcasting power is only given in "relative" terms (as in "very low, low, medium, strong") – I will have to find a way to translate this into numbers
- Buildings, mountains and other large structures can influence radio waves – how do I take care of that? Is it important?

## Work so far

Loaded the data into QGIS and started to play with it.

<img width="1102" alt="screen shot 2018-07-29 at 18 47 16" src="https://user-images.githubusercontent.com/658596/43371434-2074426e-9360-11e8-9785-aca79d870cf7.png">

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [x] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far

**Comment by kevinlitman-navarro** [comment]
- This is a really cool idea! I like the framing, looking at the two extreme ends of the radio wave density spectrum

- I'm not sure that I understand the map. Is each orange dot a cell tower and the green area the coverage from the tower? What does the blue mean? Could use some explanation when you do your first revision.

- I'm not familiar with Switzerland's geography, so not sure if this map is showing me lots of coverage than one would expect or less coverage than one would expect. Could you provide a map that just gives us context for reading your project?

**Comment by sarahslo** [comment]
so you want to have a color ramp of one color that goes from light to dark, from very low to strong.

can you outline for us where the buildings/people are? or find some way to show population density? that may start to solve your issue with buildings vs mountains...

**Comment by xeophin** [comment]
# Update
	
## Your project content: images/words/etc
	
![cloudmap-20180805](https://user-images.githubusercontent.com/658596/43687141-7672c774-989e-11e8-926c-93621eb76ddf.png)

  
## Any changes in direction or topic?

Not ... directly. I just changed the visualisation form. The broadcasting towers have now a translucent gradient. Layer each of those on top of each other, and you will get an approximation of where you have the most broadcasting going on.
	
## Problems/Questions

- While I found the "definition" of the broadcasting power categories, they were not that helpful, as they just give the magnitude
- It still does not give me the broadcasting range. Even without taking the landscape into account, this all depends on the broadcasting power as well as the broadcasting frequency, which I again don't really have.

## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [x] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by xeophin** [comment]
# Update
	
## Your project content: images/words/etc

I decided to take the three different technologies apart.

### GSM
![cloudmap-gsm-20180807](https://user-images.githubusercontent.com/658596/43807149-20f62d6c-9a75-11e8-8ff2-29eba5f4f111.png)

### UMTS
![cloudmap-umts-20180807](https://user-images.githubusercontent.com/658596/43807156-2a4e045c-9a75-11e8-9ce0-bf834bcdea02.png)

### LTE
![cloudmap-lte-20180807](https://user-images.githubusercontent.com/658596/43807163-32416834-9a75-11e8-8cbd-6393d6e42179.png)

But what does it mean?

Either GSM is in decline, only used as a backup solution – or GSM simply needs less cell towers, as it isn't designed to have as large a bandwidth as both UMTS and even more so LTE. 

  
## Any changes in direction or topic?
 
Just tried to see what happens when I take the various technologies apart – are there any obvious differences? But I guess not as much as I assumed.
	
## Problems/Questions

The data quality/source is still lacking.
	
## Checklist
	
- [ ] I have included my visuals
- [ ] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by sarahslo** [comment]
i like this approach. we need a key, and i'd suggest you make each technology a different color, signals that they are different data. can you annotate the highest number in each set? or top 3?

**Comment by julialedur** [comment]
Very cool project! I really like your maps. I agree with @sarahslo, you should make each map in a different color and add some annotations of the places with the highest broadcasting activity and maybe the ones with the lowest too if there's an interesting explanation for that. 

I also think you could get rid of some of the city labels and leave only the most important ones. I feel like right now they are creating a little bit of visual pollution in your maps.

I'm not very familiar with those technologies, so I think it would be nice to have some sort of explanation of what each of them means. But I guess you can do that in your article too. 

Looking forward to checking out your next update! :)

**Comment by xeophin** [comment]
# Final

## Project visuals/text

![screenshot_2018-08-12 the elusive quiet spot mobile phone antenna distribution in switzerland](https://user-images.githubusercontent.com/658596/44007093-7bcc9064-9e5d-11e8-8c49-0d4ad7363fae.jpg)


## Details

**Headline:** The Elusive Quiet Spots

**Published website version:** https://xeophin.github.io/lede-data-studio/antennas/

**Code repository:** https://github.com/xeophin/lede-data-studio/tree/master/code/02-antennas

**Final data set(s):** https://github.com/xeophin/lede-data-studio/blob/master/code/02-antennas/cell_phone_antennas.qgz

## What did you find to be the most difficult part of this project?

Getting anything out of the data at all.

## Are you satisfied with what you produced? Is there anything you would like to change or improve?

Obviously, the data was bad to begin with, so the final visualisation is (quite deliberately) fuzzy. The next step would be to find population density data and compare it to this data set – they will likely be similar. Also, it would be interesting to see whether it's possible to get more detailed data about it.

## Checklist

- [ ] I have included my visuals
- [ ] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
